cypress/e2e/auth_spec/redirect.cy.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference types="cypress" />
+
+import { cy, describe, it, beforeEach } from "local-cypress";
+import LoginPage from "../../pageobject/Login/LoginPage";
+
+describe("redirect", () => {
+  const loginPage = new LoginPage();
+
+  beforeEach(() => {
+    cy.log("Logging in the user devdistrictadmin:Coronasafe@123");
+  });
+
+  it("Check if login redirects to the right url", () => {
+    cy.awaitUrl("/resource/board", true);
+    loginPage.loginManuallyAsDistrictAdmin();
+    loginPage.CheckIfLoggedIn();
+    cy.url().should("include", "/resource/board");
+  });
+
+  it("Check if the redirect param works", () => {
+    cy.awaitUrl("login?redirect=http://localhost:4000/resource/board", true);
```

Comment:
I see that the current origin is hardcoded as `http://localhost:4000` , This works fine but I believe we should instead use the Cypress' `baseUrl` property, this will ensure that if in future someone decides to run tests on a different baseUrl, the tests will not malfunction. 

Reference: https://filiphric.com/cypress-basics-using-baseurl#making-assertions-with-baseurl

cc: @nihal467

Reply:
Addressed this.